aws-cs-lambda/dotnetLambda/src/dotnetLambda/Function.cs

```diff
@@ -0,0 +1,27 @@
+using System;
```

Comment:
These files are nested too deep to my liking. I would just put them directly to the `dotnetLambda` folder.